### PR TITLE
feat: followed artist artworks rail

### PR DESCRIPTION
**Pull request by dzucconi**
The type of this PR is: **Feature**

This PR resolves [FX-2212]

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))

[FX-2212]: https://artsyproduct.atlassian.net/browse/FX-2212

### CODE REVIEW

File: src/lib/Scenes/Fair2/Components/Fair2FollowedArtists.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        keyExtractor={(item, index) => String(item.artwork!.id || index)}
+        renderItem={({ item: { artwork } }) => (
+          <ArtworkTileRailCard
+            artwork={artwork!}
+            onPress={() => {
+              if (!artwork!.slug) {
+                return
+              }
+
+              return SwitchBoard.presentNavigationViewController(navRef.current, `/artwork/${artwork!.slug}`)
```
Comment (sweir27) [inline src/lib/Scenes/Fair2/Components/Fair2FollowedArtists.tsx:50]: Mind using the new `navigate` helper?

Once imported, I think it's as simple as `navigate(<url>)` and you can get rid of the ref!

File: src/lib/Scenes/Fair2/Fair2.tsx [hunk 2]
```diff
@@ -70,6 +72,7 @@ export const Fair2: React.FC<Fair2Props> = ({ fair }) => {
     "fairHeader",
     ...(hasArticles ? ["fairEditorial"] : []),
     ...(hasCollections ? ["fairCollections"] : []),
+    ...(hasFollowedArtistArtworks ? ["fairFollowedArtists"] : []),
```
Comment (sweir27) [inline src/lib/Scenes/Fair2/Fair2.tsx:75]: Mind adding a corresponding test to verify this component shows up when intended?

File: src/lib/Scenes/Fair2/Components/Fair2FollowedArtists.tsx (hunk 1 above)
Comment (dzucconi) [reply src/lib/Scenes/Fair2/Components/Fair2FollowedArtists.tsx:50]: Ah was wondering, great.

**Comment by admbtlr** [comment]
Maybe change the PR title to `feat: followed artists rail in fairs`?

File: src/lib/Scenes/Fair2/Components/Fair2FollowedArtists.tsx (hunk 1 above)
Comment (dzucconi) [reply src/lib/Scenes/Fair2/Components/Fair2FollowedArtists.tsx:50]: oh forgot this one

**Pull request merged**